WrightTools/kit.py

```diff
@@ -1153,36 +1152,43 @@ def array2string(array, sep='\t'):
     return string
 
 
-def flatten_list(l):
-    """Flatten an irregular list.
+def flatten_list(items, seqtypes=(list, tuple), in_place=True):
```

Comment:
Do we wish for the default to be in place? This is not consistent with previous behavior.

Reply:
yeah, I thought about that and I think I do want to move to in place